docs/website/content/v0.7/en/configuration/v1alpha1.md

```diff
@@ -643,6 +643,19 @@ adminKubeconfig:
 
 ```
 
+#### scheduleOnMasters
+
+Indicates if master nodes should have taint NoSchedule.
```

Comment:
my English is bad, but the phrase here seems to be confusing,  the meaing should be that`scheduleOnMasters` disables `NoSchedule` on masters.

@andrewrynhard help please :)

Reply:
It isn't too bad, but I think it could be better. I will propose making `scheduleOnMasters` `allowSchedulingOnMasters`, and then the description could be:

```
Indicates if master nodes are schedulable.
```

@rsmitty @Ulexus feel free to chime in.